network/discovery/options.go

```diff
@@ -80,10 +80,13 @@ func (opts *DiscV5Options) IPs() (net.IP, net.IP, string) {
 	return ipAddr, bindIP, n
 }
 
+var pid = [6]byte{'s', 's', 'v', 'd', 'v', '5'}
+
 // DiscV5Cfg creates discv5 config from the options
 func (opts *DiscV5Options) DiscV5Cfg(logger *zap.Logger) (*discover.Config, error) {
 	dv5Cfg := discover.Config{
-		PrivateKey: opts.NetworkKey,
+		PrivateKey:   opts.NetworkKey,
+		V5ProtocolID: &pid, // TODO: REVERT!
```

Comment:
you want to revert or leave this?

Reply:
will revert, its just to be able to test on stage env